src/DownloadButton/index.js

```diff
@@ -4,7 +4,7 @@ import styled, { css } from 'styled-components'
 import isClient from '../utils/isClient'
 import { logEvent } from '../utils/ga'
 
-const VERSION = `0.40.1`
+const VERSION = `0.40.2`
```

Comment:
merge or rebase on top of master?

Reply:
Yeah Idk what happened there. Looks like I accidentally cherry picked a commit?? Weird. I removed it now (had to force-push to my fork).